samples/micrometer-samples-core/src/main/java/io/micrometer/core/samples/utils/SampleRegistries.java

```diff
@@ -169,6 +169,18 @@ public Duration step() {
             public String get(String k) {
                 return null;
             }
+
+            public boolean enableAggregation() {
```

Comment:
Why is this change required?

Reply:
I don't believe this is anymore 👍 removing